core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -235,20 +235,20 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     }
 
     ExecutionServiceWorkflowOptions(
-      s"gs://${workspace.bucketName}/${submissionId}",
+      submission.outputPath,
```

Comment:
I think this is the gcs root you're changing, not (just) the outputs path. So I think what this will do is let users change their workflow execution root, rather than let them move outputs files away from the other junk generated by workflow execution

Reply:
This seems like a terminology mistake on my part. I was treating the phrases "outputs" and "everything written by Cromwell during execution" as the same thing, but in Cromwell-world it seems like those two are distinct and separate terminology? What we are most interested in doing is allowing the user to specify a path for execution so they can avoid cluttering their workspace bucket in Terra, it's less important to specify a path to copy the "good" files to, leaving the junk behind. I do think exposing more of the Cromwell workflow options in Terra UI would be useful in the future, but I wouldn't want to add that to the scope of this PR. Terra UI also currently refers to the execution path as the "outputs" path, so we may need to change that to be more accurate as well.

To be more accurate with terminology, how does it sound if we reframe this PR as "Allow custom execution path for submissions"? If that works for you, I will go in and update the code accordingly to reflect that.

Reply:
Aha, yes, and that makes a lot more sense. Yes, "execution path", "execution root", "root execution directory", "submission execution directory" or even "submission directory" all seem like fine options

Reply:
Done! Renamed everything from outputPath -> executionPath in https://github.com/broadinstitute/rawls/pull/1896/commits/87f5d429e93a2ce59ef26bdfe8ad80c17c7b679b